modules/test/playwright/tests/osb-faro-web/acquisition.spec.ts

```diff
@@ -0,0 +1,474 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {navigateToACSitesPageViaURL} from './utils/navigation';
+import {CardSelector} from './utils/selectors';
+import {closeSessions} from './utils/sessions';
+import {changeTimeFilter} from './utils/time-filter';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+async function sendEventByURL(page: Page, queryParams: string) {
+	await page.goto(liferayConfig.environment.baseUrl + `/home?${queryParams}`);
+
+	await page.waitForTimeout(3000);
+}
+
+async function checkAcquisitionChannelCount(
+	acquisitionChannel: string,
+	count: string,
+	page: Page
+) {
+	const acquisitionChannelElement = page.getByText(acquisitionChannel);
+
+	await expect(acquisitionChannelElement).toBeVisible({
+		timeout: 5 * 1000,
+	});
+
+	const acquisitionChannelCount = await page.evaluate(
+		(acquisitionChannel) => {
+			const acquisitionChannelRow = Array.from(
+				document.querySelectorAll('.acquisitions-card-root tbody tr')
+			).find(
+				(element) =>
+					element.querySelector('.table-title').textContent ===
+					acquisitionChannel
+			);
+
+			return acquisitionChannelRow.querySelector('.count').textContent;
+		},
+		acquisitionChannel
+	);
+
+	expect(acquisitionChannelCount).toBe(count);
+}
+
+test('check if acquisition card displays PAID SEARCH channel after receiving an event', async ({
+	apiHelpers,
+	page,
+}) => {
+	const channelName = 'My Property - ' + getRandomString();
+
+	const {channel, project} = await syncAnalyticsCloud({
+		apiHelpers,
+		channelName,
+		page,
+	});
+
+	await test.step('send event to initialize channel followed by closing the session', async () => {
+		await sendEventByURL(page, 'utm_medium=paidsearch');
+
+		await closeSessions(apiHelpers, page);
+	});
+
+	await test.step('go to AC workspace', async () => {
+		await navigateToACSitesPageViaURL({
+			channelID: channel.id,
+			page,
+			projectID: project.groupId,
+		});
+
+		await page.waitForTimeout(3000);
```

Comment:
Hey @fmarcio, Is this pause really necessary? I think you could remove it or if necessary, then put some `waitForSelector` to ensure that the AC has been loaded before executing the next step within the `navigateToACSitesPageViaURL` function

Reply:
@ViniLopes20 done